app/views/bill-runs/charge-reference-details.njk

```diff
@@ -66,7 +66,8 @@
       <span class="inline govuk-!-margin-right-3">
         {{ govukButton({
           text: "Change the authorised volume",
-          classes: "govuk-button--secondary"
+          classes: "govuk-button--secondary",
+          href: chargeReference.id + '/amend-authorised-volume'
```

Comment:
I think the norm is to add `preventDoubleClick: true` to the button

Reply:
Good spot, thank you. Ive missed this out in a few places. I have created a [new PR](https://github.com/DEFRA/water-abstraction-system/pull/1050) to make this consistent.